pkg/virtual-clusters/components/InstallK3k.vue

```diff
@@ -45,27 +45,26 @@ export default {
 
     k3kInstalled: {
       type:    Boolean,
-      default: true
+      default: false
+    },
+
+    showButtonOnly: {
+      type:    Boolean,
+      default: false
     }
+  },
 
+  data() {
+    // track if k3k chart is present in the currently selected host cluster AND
+    // track if the user installed k3k while viewing this page
+    return { didInstallK3k: false, localParentCluster: this.parentCluster };
```

Comment:
I'm storing the value of `parentCluster` locally so that this component can immediately update what is displayed in the dropdown, but determine whether or not k3k is installed in the newly selected cluster before emitting a new `parentCluster` value and triggering re-computation in the ClusterPolicy component: this ensures the ClusterPolicy component has up-to-date information about k3k installation before attempting to fetch policies or showing the user errors about not being able to fetch policies.